include/open62541/client.h

```diff
@@ -147,6 +147,10 @@ typedef struct {
     /* Certificate Verification Plugin */
     UA_CertificateVerification certificateVerification;
 
+    /* Available SecurityPolicies for Authentication. */
+    size_t authSecurityPoliciesSize;
+    UA_SecurityPolicy *authSecurityPolicies;
```

Comment:
Please document when a certificate authentication is used.
If there is a (matching) authSecurityPolicy, do we automatically use that for authentication?
Or does that have to be configured in addition in the userTokenPolicy?